registry/donor/models.py

```diff
@@ -44,6 +44,21 @@ def from_list(cls, list):
             donation_count=list[8],
         )
 
+    def as_original(self):
+        fields = [
+            "rodne_cislo",
+            "first_name",
+            "last_name",
+            "address",
+            "city",
+            "postal_code",
+            "kod_pojistovny",
+            "donation_count",
+        ]
+        line = ";".join([str(getattr(self, field)) for field in fields])
```

Comment:
A co když nějaké z polí obsahuje středník?

Reply:
Prý to není možné, protože takový záznam by neprošel importem. Máme na to test?

Reply:
Ano, máme testy pro situace kde je položek na řádku málo, moc i kde je jen zdvojený středník, přičemž to poslední jsme schopni sami opravit. Udělám ještě PR, kde ty testy trošku rozšířím.